### PR TITLE
Split out 2022 changes from the changelog

### PR DESCRIPTION
**Pull request by MadLittleMods**
Split out 2022 changes from the changelog so the rendered version in GitHub doesn't timeout as much.

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog).
* [ ] ~~Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)~~
* [x] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

File: docs/changelogs/CHANGES-2022.md [hunk 1]
```diff
@@ -0,0 +1,2766 @@
...
+
+Bugfixes
+--------
+
+- Fix a long-standing bug where relations from other rooms could be included in the bundled aggregations of an event. ([\#11516](https://github.com/matrix-org/synapse/issues/11516))
+- Fix a long-standing bug which could cause `AssertionError`s to be written to the log when Synapse was restarted after purging events from the database. ([\#11536](https://github.com/matrix-org/synapse/issues/11536), [\#11642](https://github.com/matrix-org/synapse/issues/11642))
+- Fix a bug introduced in Synapse 1.17.0 where a pusher created for an email with capital letters would fail to be created. ([\#11547](https://github.com/matrix-org/synapse/issues/11547))
+- Fix a long-standing bug where responses included bundled aggregations when they should not, per [MSC2675](https://github.com/matrix-org/matrix-doc/pull/2675). ([\#11592](https://github.com/matrix-org/synapse/issues/11592), [\#11623](https://github.com/matrix-org/synapse/issues/11623))
+- Fix a long-standing bug that some unknown endpoints would return HTML error pages instead of JSON `M_UNRECOGNIZED` errors. ([\#11602](https://github.com/matrix-org/synapse/issues/11602))
+- Fix a bug introduced in S
```
Comment (DMRobertson) [inline docs/changelogs/CHANGES-2022.md:2766]: I'm guessing:

```suggestion
**Changelogs for older versions can be found [here](CHANGES-2021.md).**
```

**Review by DMRobertson — APPROVED** [review]
I think this probably wants to target the release branch? 

Aside from that and the typo, LGTM

File: docs/changelogs/CHANGES-2022.md (hunk 1 above)
Comment (DMRobertson) [reply docs/changelogs/CHANGES-2022.md:2766]: (It looks like Github's `suggestion` renderer is having a bad time here)

File: changelog.d/15846.misc [hunk 2]
```diff
@@ -0,0 +1 @@
+Split out 2022 changes from the changelog so the rendered version in GitHub doesn't timeout as much.
```
Comment (MadLittleMods) [inline changelog.d/15846.misc:1]: > I think this probably wants to target the release branch?

Just an ongoing maintenance thing. Not really specific to this release. Going to opt for `develop`

(it does render sometimes)

Comment (MadLittleMods) [reply changelog.d/15846.misc:1]: hmm, I updated it to merge into the release branch in any case. Will merge the release into `develop` after as well so we can benefit in the mean-time.

**Comment by MadLittleMods** [comment]
Thanks for the review @DMRobertson 🦭

**Pull request merged**